### ISSUE TITLE
Unknown field names without arguments are transformed with a trailing space

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1203]
When an unknown field name is encountered in a python docstring, a space is appended to the field name by the sphinx.util.docfields.DocFieldTransformer. The following docstring:

```
#!python

def func_with_unknown_field():
    """Demonstrate unknown field transformation

       :Field_Name: Field Value

    """
    pass
```

Yields the following incorrect HTML for Field_Name:

```
#!html

<table rules="none" frame="void" class="docutils field-list">
  <colgroup>
    <col class="field-name">
    <col class="field-body">
  </colgroup>
  <tbody valign="top">
    <tr class="field-odd field">
      <th class="field-name">Field_name :</th>
      <td class="field-body">Field Value</td>
    </tr>
  </tbody>
</table>
```

Note the extra space in `Field_name :`.

See the [reStructuredText Markup Spec](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#field-lists) regarding field lists.

For examples of this bug in action, see the "Returns" and "Parameters" field names in the [NumPy documentation](http://docs.scipy.org/doc/numpy/reference/generated/numpy.array.html).

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1203
- Originally reported by: Rob Ruana
- Originally created at: 2013-06-23T21:57:31.802

**Comment by shimizukawa** [comment]
_From Rob Ruana on 2013-06-24 03:05:57+00:00_

Pull [request #149](https://bitbucket.org/birkenfeld/sphinx/pull-request/149) fixes this issue.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-09-16 07:18:50+00:00_

Merged in RelentlessIdiot/sphinx (https://bitbucket.org/birkenfeld/sphinx/pull-request/149)

Fix #1203: Do not append space to unknown field names.

→ <<cset 1a99c74e4b8c2b91081562db4bf7292d0b5eca9b>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-09-16 07:18:50+00:00_

Fix #1203: Does not append space to unknown field names.

If an unknown field name is encountered, a space is only
appended if the field arguments are not empty. Also,
only the first character of the field name is upper cased,
and the case of the rest of the string remains unchanged.

→ <<cset 718f4a6f8c3efcf286c79bf4cba8240247cdf206>>
